src/pages/news/News.jsx

```diff
@@ -1,16 +1,32 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import SingleNews from "./SingleNews";
 
 const News = () => {
+  const url = import.meta.env.VITE_REACT_APP_BASE_URL;
```

Comment:
The best name for this variable should be baseUrl instead. It is more descriptive.

Reply:
Okay, thanks.